book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Hi Good job so far, however please notice as mentioned in the requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/232193955-ef4240fb-f841-4525-b8cc-f8ae0678486f.png)

please make sure book has many rentals, currently you have initialised rentals but also add a method so that when we add a rental for a book, a rental is also created with book as a self parameter please see :point_down: also make sure the rentals attribute has setter and getter both as you will need it currently you only have the getter that is only readable for rentals attribute
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```